### ISSUE TITLE
Link checker as a warning

### ISSUE THREAD

**Issue by itsbenweeks** [mitodl/openedx-course-test#25]

**Comment by carsongee** [comment]
I think link checking should be failure, if there is an issue with legitimate links causing failures, we should work on fixing the checker.  For example, right now it only waits 1 second, which generally should be fine for  a HEAD call unless you are linking to something abominably slow, or that doesn't properly support HEAD.

**Comment by itsbenweeks** [comment]
I understand this belief, and generally agree with it. There are some statuses that you may want to look out for, then, since some sites may not like the agent header of the checker that is visiting their server.

**Comment by itsbenweeks** [comment]
Hmmm.. Redacted, the issue that sparked this was linking to other portions of courseware as an external link, so it was failing on auth to edx.org.

There is this other issue, though:

```
External link http://www.cambridge.org/9780521880329 in i4x://MITx/8.05x/html/staff invalid. Request exception occurred, HTTPConnectionPool(host='www.cambridge.org', port=80): Read timed out. (read timeout=1): FAIL
```

**Comment by carsongee** [comment]
yeah, I've seen that one, it takes like 2 seconds to respond to HEAD....I guess I can just dial back the timeout

**Comment by carsongee** [comment]
hmm, actually I think I'll probably just stop following redirects and mark a 302 as a success....the HEAD is quick, but there is a really long redirect chain happening.
